gradle/root/coverage.gradle

```diff
@@ -1,16 +1,19 @@
 apply plugin: 'jacoco'
-
-// TODO later include subprojects, possible ideas https://gist.github.com/aalmiray/e6f54aa4b3803be0bcac
+apply from: 'gradle/dependencies.gradle'
 
 jacoco {
-    toolVersion = "0.7.9"
+    toolVersion = libraries.jacocoVersion
```

Comment:
I'm curious why we need to specify the tool version. Can you get rid of the version and see if it works? If we bump Gradle, we generally want the best jacoco version that works with given Gradle version. Let's try if we can rely on the default.

Not blocking for this PR.

Reply:
Gradle usually has outdated version of JaCoCo by default (see https://github.com/gradle/gradle/commit/987d86f36a05b9090ab498dfbe77fe3dbc0f8e39 - 6 months after the release of 0.7.9 - thanks to the PR from the JaCoCo committer). In addition Mockito tended (in the past) to do not use the newest Gradle version.
Therefore, it's redundant right now, but it can be needed once 0.7.10 is released. As a result, having it just one place would be probably enough to be able to uncomment one line in `coverage.gradle`.